lib/src/common/model/activity.dart

```diff
@@ -1,42 +1,51 @@
 part of github.common;
 
 /// Model class for an event.
+@JsonSerializable(createToJson: false)
 class Event {
+  @JsonKey()
   Repository repo;
 
-  User actor;
+  @JsonKey()
+  Actor actor;
```

Comment:
Is it possible to keep this as a User object? This was it wouldn't be a breaking change. 
I see you added a display_login field to Actor but I can't find any reference to that being in the response anywhere.
Is that something new?